lib/beaker/options/parser.rb

```diff
@@ -7,7 +7,7 @@ class Parser
       GITREPO      = 'git://github.com/puppetlabs'
       #These options can have the form of arg1,arg2 or [arg] or just arg,
       #should default to []
-      LONG_OPTS    = [:helper, :load_path, :tests, :pre_suite, :post_suite, :install, :pre_cleanup, :modules]
+      LONG_OPTS    = [:helper, :load_path, :tests, :pre_suite, :provision_state, :post_suite, :install, :pre_cleanup, :modules]
```

Comment:
should this say `:preserve_state`?

Reply:
oh, definitely.... thanks for catching that!